distribution/lib/Standard/Table/0.0.0-dev/src/Internal/Widget_Helpers.enso

```diff
@@ -93,15 +93,28 @@ make_column_name_multi_selector table display:Display=Display.Always add_regex:B
 
 ## PRIVATE
    Make a column reference by name selector.
-make_column_ref_by_name_selector : Table -> Display -> Boolean -> Boolean -> Boolean -> Boolean -> Widget
-make_column_ref_by_name_selector table display:Display=Display.Always add_text:Boolean=False add_regex:Boolean=False add_number:Boolean=False add_boolean:Boolean=False =
+make_column_ref_by_name_selector : Table -> Display -> Boolean -> Boolean -> Boolean -> Boolean -> Boolean -> Widget
+make_column_ref_by_name_selector table display:Display=Display.Always add_text:Boolean=False add_regex:Boolean=False add_number:Boolean=False add_boolean:Boolean=False add_named_pattern:Boolean=False =
     text = if add_text then [Option "<Text Value>" "''"] else []
     regex = if add_regex then [Option "<Regular Expression>" "(regex '')"] else []
     number = if add_number then [Option "<Number Value>" "0"] else []
     boolean = if add_boolean then [Option "<True/False>" "True"] else []
+    named_pattern = if add_named_pattern.not then [] else
+        Vector.build builder->
+            builder.append (Option "<Leading_Whitespace>" "Named_Pattern.Leading_Whitespace")
+            builder.append (Option "<Trailing_Whitespace>" "Named_Pattern.Trailing_Whitespace")
+            builder.append (Option "<All_Whitespace>" "Named_Pattern.All_Whitespace")
+            builder.append (Option "<Leading_Numbers>" "Named_Pattern.Leading_Numbers")
+            builder.append (Option "<Trailing_Numbers>" "Named_Pattern.Trailing_Numbers")
+            builder.append (Option "<Non_ASCII>" "Named_Pattern.Non_ASCII")
+            builder.append (Option "<Tabs>" "Named_Pattern.Tabs")
+            builder.append (Option "<Letters>" "Named_Pattern.Letters")
+            builder.append (Option "<Numbers>" "Named_Pattern.Numbers")
+            builder.append (Option "<Punctuation>" "Named_Pattern.Punctuation")
+            builder.append (Option "<Symbols>" "Named_Pattern.Symbols")
```

Comment:
Feels odd that we angle bracket on this one and not in the Text ones but guess ok.

Reply:
Yeah... I struggled with angle brackets or not. I'm not overly happy with it. If someone has a better idea I'm happy to change it. Angle brackets everywhere?